internal/saucecloud/cloud.go

```diff
@@ -140,7 +140,9 @@ func (r *CloudRunner) runJob(opts job.StartOptions) (j job.Job, interrupted bool
 	}
 
 	jobDetailsPage := fmt.Sprintf("%s/tests/%s", r.Region.AppBaseURL(), id)
-	log.Info().Str("suite", opts.Suite).Str("url", jobDetailsPage).Msg("Suite started.")
+	log.Info().Str("suite", opts.Suite).Str("url", jobDetailsPage).
+		Str("device", opts.DeviceName).Str("platform", opts.PlatformName).
+		Str("name", opts.Name).Msg("Suite started.")
```

Comment:
@tianfeng92 `device` wont we a value for cypress. On that platform we probably want to show the `browser`.